src/components/va-alert/va-alert.css

```diff
@@ -13,7 +12,7 @@
 }
 
 ::slotted([slot='headline']) {
-  font-size: 1em !important;
+  font-size: 2rem !important;
```

Comment:
It looks like this does the trick.

Testing sample:

```jsx
        <va-alert status="success">
          <h4 slot="headline">I am a level 4 headline</h4>
          <div>Content blah blah blah</div>
        </va-alert>
        <br />
        <va-alert status="success">
          <h3 slot="headline">Congrats, you did it!</h3>
          <div>Content blah blah blah</div>
        </va-alert>
        
        <AlertBox headline="Testing" status="info">
          <div>Some more content</div>
        </AlertBox>
 ```

In firefox:

![image](https://user-images.githubusercontent.com/2008881/124316470-7e5c0000-db2a-11eb-8727-dd14b62889b5.png)


In IE11:

![signal-2021-07-02-114443](https://user-images.githubusercontent.com/2008881/124316843-13f78f80-db2b-11eb-8825-294e9fd992fe.png)